=== modified file 'src/demux.py'
--- src/demux.py	2009-03-19 17:16:36 +0000
+++ src/demux.py	2009-04-21 14:01:30 +0000
@@ -48,8 +48,8 @@
         self.clients = {'BEE2': AgentClient(Bee2Agent, host = 'bee2',
                                             port = 8915)
                         , 'DAQ': DaqAgent()
-                        , 'POWER': PowerstripAgent()
-                        , 'SYNTH': SynthAgent()
+                        # , 'POWER': PowerstripAgent()
+                        # , 'SYNTH': SynthAgent()
                         }
 
     def get(self, keys = index):
@@ -63,17 +63,17 @@
         """
         if keys == index:
             return ['BEE2/' + key for key in self.clients['BEE2'].get(index)] +\
-                   ['DAQ/' + key for key in self.clients['DAQ'].get(index)] +\
-                   ['POWER/' + key for key in self.clients['POWER'].get(index)] +\
-                   ['SYNTH/' + key for key in self.clients['SYNTH'].get(index)]
+                   ['DAQ/' + key for key in self.clients['DAQ'].get(index)] # +\
+                   # ['POWER/' + key for key in self.clients['POWER'].get(index)] +\
+                   # ['SYNTH/' + key for key in self.clients['SYNTH'].get(index)]
         elif keys == bee2_index:
             return ['BEE2/' + key for key in self.clients['BEE2'].get(index)]
         elif keys == daq_index:
             return ['DAQ/' + key for key in self.clients['DAQ'].get(index)]
-        elif keys == power_index:
-            return ['POWER/' + key for key in self.clients['POWER'].get(index)]
-        elif keys == synth_index:
-            return ['SYNTH/' + key for key in self.clients['SYNTH'].get(index)]
+        # elif keys == power_index:
+        #     return ['POWER/' + key for key in self.clients['POWER'].get(index)]
+        # elif keys == synth_index:
+        #     return ['SYNTH/' + key for key in self.clients['SYNTH'].get(index)]
 
         # HACK:
         # try out combining BEE2 and DAQ calls
@@ -108,18 +108,18 @@
         bee2_values = self.clients['BEE2'].get(bee2_keys)
         daq_keys = [key.replace('DAQ/', '') for key in client_keys['DAQ']]
         daq_values = self.clients['DAQ'].get(daq_keys)
-        power_keys = [key.replace('POWER/', '') for key in client_keys['POWER']]
-        power_values = self.clients['POWER'].get(power_keys)
-        synth_keys = [key.replace('SYNTH/', '') for key in client_keys['SYNTH']]
-        synth_values = self.clients['SYNTH'].get(synth_keys)
+        # power_keys = [key.replace('POWER/', '') for key in client_keys['POWER']]
+        # power_values = self.clients['POWER'].get(power_keys)
+        # synth_keys = [key.replace('SYNTH/', '') for key in client_keys['SYNTH']]
+        # synth_values = self.clients['SYNTH'].get(synth_keys)
         none_keys = client_keys['']
         none_values = ['KeyError' for i in range(len(none_keys))]
 
         # build result
         client_values = {'BEE2': bee2_values,
                          'DAQ': daq_values,
-                         'POWER': power_values,
-                         'SYNTH': synth_values,
+                         # 'POWER': power_values,
+                         # 'SYNTH': synth_values,
                          '': none_values}
 
         result = []

